### DISCUSSION TITLE
GitHub PassKey registration failed when use chrome devtools webAuthn

### DISCUSSION THREAD

**Discussion by feifei08** [community/community#66459] [New to GitHub]
I try to register a PassKey for my github account, but i didn't have an authenticator.
Then i found that the chrome devtools provides a virtual authenticator to verify webAuthn API, so i tried to use chrome devtools webAuthn as an authenticator to register PassKey.  But github prompts "github.com may need to use a updated or different types of device".

My question is: 
    1、Is anyone successfully registered Passkey? 
    2、Which brand and type of authenticator is used? 
    3、Why can't Chrome DevTools Webauthn be supported?

![image](https://github.com/community/community/assets/144195649/872be830-35ce-4cf7-9b90-e8705c709788)

**Comment by queenofcorgis** [comment]
👋🏼 👋🏼 @feifei08, 

Thanks for taking the first steps to keep your account safe and checking out passkeys.

1. Yes, we do have folks who we're able to successfully register during this public beta
2. GitHub does not require a specific authenticator app - it is your choice! 
3. The error message you're seeing is due to either missing the resident key or user verification requirement which are required. In your case, it looks like you're missing both. I would first try filling in those missing bits as a next step.
